connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -105,6 +105,7 @@ public class MirrorSourceConnector extends SourceConnector {
     private int replicationFactor;
     private Admin sourceAdminClient;
     private Admin targetAdminClient;
+    private boolean heartbeatsReplicationEnabled = true;
```

Comment:
Why do you set this at all when in start() you get the actual config value (or default) anyways?

Reply:
I'm setting this to keep the old behavior in unit tests - most unit tests do not call start, but use one of the ctor overloads. If I don't set it to true explicitly, I need to make sure that all of those ctor overloads set it to true, but this seemed like a simpler solution.

Reply:
Well, I see your point but it seems like the test constructors are quite inconsistent in this and they pick and choose in what they initialize. I ran all tests in `org.apache.kafka.connect.mirror` without setting this and they seemed to work so I'm not sure if there are any effects on existing tests.
It would be nice to have a constructor "chain" that gives everything some default values but that would be a different work.
I would prefer if we kept the convention of initializing in the constructor/start() when needed but if you insist on this, then please use the constant`HEARTBEATS_REPLICATION_ENABLED_DEFAULT` in `MirrorSourceConfig` to avoid hardcoding values, that would be a compromise I could live with :).

Reply:
okay, I removed the initialization